srcpkgs/Carla/patches/pyliblo3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/source/frontend/carla_host_control.py b/source/frontend/carla_host_control.py
+index 00c6958..2d5d4da 100755
+--- a/source/frontend/carla_host_control.py
++++ b/source/frontend/carla_host_control.py
+@@ -32,7 +32,7 @@ from carla_host import *
+ # ------------------------------------------------------------------------------------------------------------
+ # Imports (liblo)
+ 
+-from liblo import (
++from pyliblo3 import (
+   Address,
+   AddressError,
+   ServerError,
```

Comment:
Use a properly formtted patch for the upstream commit, so that we can track provenance of our patches.